rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/start_scan/action.py

```diff
@@ -0,0 +1,52 @@
+import insightconnect_plugin_runtime
+from .schema import StartScanInput, StartScanOutput, Input, Component
+
+# Custom imports below
+from insightconnect_plugin_runtime.exceptions import PluginException
+from ipaddress import ip_address, IPv4Address, IPv6Address
+
+
+def format_body(hostnames: [], ips: []):
+    asset_body = ""
+    for hostname in hostnames:
+        asset_body = asset_body + "asset.name STARTS WITH '" + hostname + "' || "
+    for ip in ips:
+        if type(ip_address(ip)) is IPv4Address:
+            asset_body = asset_body + "asset.ipv4 = " + ip + " || "
+        if type(ip_address(ip)) is IPv6Address:
+            asset_body = asset_body + "asset.ipv6 = " + ip + " || "
+    if asset_body[len(asset_body) - 2] == "|":
+        asset_body = asset_body[: len(asset_body) - 4]
+    body_object = {"asset": asset_body}
+    return body_object
```

Comment:
Can you make this a private function (prefix with _) and place it below the class? Additionally, can you make the type hints for the arguments more specific? If they are string lists, for example, use `[str]`. Also adding a return hint would be helpful (example `def some_func() -> str:`)